service/src/main/java/uk/nhs/adaptors/oneoneone/cda/report/mapper/CompositionMapper.java

```diff
@@ -110,11 +124,48 @@ public Composition mapComposition(POCDMT000002UK01ClinicalDocument1 clinicalDocu
             }
         }
 
+        //TODO - add list of careplans into sections above, update tests also
+        List<DomainResource> resources = new ArrayList<>(carePlans);
+        for (Reference reference : encounter.getIncomingReferral()) {
+            if (reference.getResource().fhirType().equals("ReferralRequest")) {
```

Comment:
Use `ResourceType.ReferralRequest.name()` instead of string `"ReferralRequest"`

Reply:
Done